js/private/node-patches/src/fs.cts

```diff
@@ -283,18 +306,8 @@ export function patcher(fs: any = _fs, roots: string[]) {
                     ) {
                         return cb(null, next)
                     }
-                    // The escape from the root is not mappable back into the root; we must make
```

Comment:
Without this change some of the non-`Sync` `js/private/test/node-patches/readlink.js` tests fail. I did this by copy+pasting the `Sync` version, then changing it back callbacks and non-`Sync`, this block of deleted code did not exist in the `Sync` version and I don't know why...